CGATPipelines/pipeline_enrichment/pipeline.ini

```diff
@@ -22,6 +22,73 @@ version=0.1
 
 # The full version, including alpha/beta/rc tags.
 release=0.1
+
+
+
+############################################################
+# Options for running each enrichment analysis
+############################################################
+[analysis]
+# Analysis type, options are:
+# GSEA (Geneset enrichment analysis) and
+# ORA  (Over Representative Analysis)
+gsea=1
+ora=1
+
+##########################################################
+# Parameters for GSEA analysis.
+
+
+[id_gsea]
+# id type of input gene list.
+type=entrez
+# which gene id type is used for the analysis
+# can be ensemblg, entrez or symbol
+# The analysis will be based on this ID type 
+# for symbol, append the organism id of the species.
+# e.g. symbol_Homo_sapiens, symbol_Mus_musculus
+## organism id in entrez - 9606 = human, 10090 = mouse
+# others here: http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/
+# NOTE:id type of gene list and gene set should be same. This option is 
+# only for translating gene list id to your required id type.
+to_convert=entrez
+
+[geneset]
+# path to geneset
+name=/ifs/projects/proj073/GSEA_PIPELINE/Gene_set.gmt
+
+[db_gsea]
+# path to a annotated database generated using pipeline_geneinfo.py
+name=/ifs/projects/reshma/DATABASE/human_db_110817
+
+[stats_gsea]
+# details of statistical tests for enrichment
+
+# Minimum size of a gene set.Gene sets smaller than this are excluded 
+# from the analysis
+min_size=25
+
+# Maximum size of a gene set. Gene sets larger than this are excluded from the analysis
+max_size=500
+
+# A number use to initialize a pseudorandom number generator
+seed=42
+
+# Number of permutations to perform in assessing the statistical significance of the enrichment score 
+permut=10
+
+# Number of gene sets to display enrichment plots (plots will be displayed for the specified 
+# number of gene sets with the highest absolute normalized enrichment scores (each phenotype) 
+display_num=5
+
+# Number of top scoring gene sets, which is used for leading edge analysis
+ngeneset=10
+###########################################################
+
+
+
+###########################################################
+# Parameters for ORA analysis
 [db]
 # path to a database generated using pipeline_geneinfo.py
 name=/ifs/projects/katherineb/pipeline_enrichment_testing/OO2/humannew
```

Comment:
Although not updated in this pull request, is `humannew` a file that should be located in a user independent folder?